server/lib/pbench/__init__.py

```diff
@@ -153,16 +154,25 @@ def get_pbench_logger(caller, config):
         except FileExistsError:
             # directory already exists, ignore
             pass
-        fh = logging.FileHandler(os.path.join(logdir, "{}.log".format(caller)))
-        fh.setLevel(logging.DEBUG)
+
+        if config.logger_type == "file":
+            handler = logging.FileHandler(os.path.join(logdir, "{}.log".format(caller)))
+        elif config.logger_type == "devlog":
+            handler = logging.SyslogHandler(address='/dev/log', facility=logging.handlers.SysLogHandler.LOG_DAEMON, socktype=socket.SOCK_STREAM)
```

Comment:
The SOCK_STREAM implementation seems to be buggy. Just delete the `socktype` option both here and on line 163.